libvmi/events.h

```diff
@@ -33,7 +33,7 @@
 #ifndef LIBVMI_EVENTS_H
 #define LIBVMI_EVENTS_H
 
-#define VMI_EVENTS_VERSION 0x00000002
+#define VMI_EVENTS_VERSION 0x00000003
```

Comment:
PR description already stated "Bumping events version as this is an ABI change."